chess-AI/mcts.py

```diff
@@ -1,8 +1,11 @@
 from math import sqrt
-
```

Comment:
These blank lines should be kept. Should have a blank line between each "type" of import; builtin includes at the top; 3rd party includes after one newline; project specific includes after one newline. See [style guide](https://google.github.io/styleguide/pyguide.html#313-imports-formatting)